system/Debug/Toolbar/Collectors/Database.php

```diff
@@ -138,59 +138,13 @@ protected function formatTimelineData(): array
 	 */
 	public function display(): array
 	{
-		// Key words we want bolded
-		$highlight = [
-			'SELECT',
-			'DISTINCT',
-			'FROM',
-			'WHERE',
-			'AND',
-			'LEFT&nbsp;JOIN',
-			'RIGHT&nbsp;JOIN',
-			'JOIN',
-			'ORDER&nbsp;BY',
-			'GROUP&nbsp;BY',
-			'LIMIT',
-			'INSERT',
-			'INTO',
-			'VALUES',
-			'UPDATE',
-			'OR&nbsp;',
-			'HAVING',
-			'OFFSET',
-			'NOT&nbsp;IN',
-			'IN',
-			'LIKE',
-			'NOT&nbsp;LIKE',
-			'COUNT',
-			'MAX',
-			'MIN',
-			'ON',
-			'AS',
-			'AVG',
-			'SUM',
-			'(',
-			')',
-		];
-
-		$data = [
-			'queries' => [],
-		];
-
-		foreach (static::$queries as $query)
-		{
-			$sql = $query->getQuery();
-
-			foreach ($highlight as $term)
-			{
-				$sql = str_replace($term, "<strong>{$term}</strong>", $sql);
-			}
-
-			$data['queries'][] = [
+		$data['queries'] = array_map(function ($query) {
+			return [
+				//@phpstan-ignore-next-line
 				'duration' => ($query->getDuration(5) * 1000) . ' ms',
```

Comment:
I'm not sure of the phpstan error you're suppressing but if we do it like this will it be gone?
```suggestion
		$data['queries'] = array_map(function (Query $query) {
			return [
				'duration' => (string) ($query->getDuration(5) * 1000) . ' ms',
```

Reply:
PhpStan error happens at
$query->getDuration(5) * 1000

since getDuration() returns string and it warns about doing multiplication on the string so it will be more appropriate to do
floatval($query->getDuration(5))*1000